client/blocks/inline-help/style.scss

```diff
@@ -70,6 +71,20 @@
 		background: $blue-medium;
 		border-color: darken( $blue-medium, 5 )
 	}
+
+	&.has-notification:before {
+		display: block;
+		background-color: $orange-jazzy;
+		border: 2px solid #fff;
+		content: "";
+		border-radius: 50%;
+		width: 10px;
+		height: 10px;
+		position: absolute;
+		top: -2px;
+		left: -3px;
+		z-index: 2;
```

Comment:
We should use our `z-index` function here: https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md